internal/gcs/container.go

```diff
@@ -87,12 +117,18 @@ func (c *Container) IsOCI() bool {
 	return c.gc.os != "windows"
 }
 
-// Close releases associated with the container.
+// Close releases resources associated with the container, but does not terminate
+// the container on the guest, or wait for it.
 func (c *Container) Close() error {
+	_, span := trace.StartSpan(context.Background(), "gcs::Container::Close")
+	defer span.End()
+	span.AddAttributes(trace.StringAttribute(logfields.ContainerID, c.id))
+
```

Comment:
Perhaps the span was only in the sync.Once block as this can be called multiple times?

Reply:
But the sync.Once also did no other work inside which is just odd..

Reply:
I figured if someone called it and did not see a span created, they would assume that code path wasnt reached.
Plus, it feels weird to put logging and tracing code inside of a `Once`